antbs/database/transaction.py

```diff
@@ -21,23 +21,28 @@
 # along with AntBS; If not, see <http://www.gnu.org/licenses/>.
 
 
-import os, subprocess, tempfile, time
-import shutil
 import datetime
+import os
+import shutil
+import subprocess
+import tempfile
+import time
 from multiprocessing import Process
+
```

Comment:
Imports can be sorted.

*PyImportSortBear, severity NORMAL, section `python`.*

The issue can be fixed by applying the following patch:

```diff
--- a/antbs/database/transaction.py
+++ b/antbs/database/transaction.py
@@ -29,11 +29,10 @@
 import time
 from multiprocessing import Process
 
+import utils.docker_util as docker_util
 from pygments import highlight
 from pygments.formatters import HtmlFormatter
 from pygments.lexers import BashLexer
-
-import utils.docker_util as docker_util
 from transaction_handler import logger
 from utils.logging_config import logger
 from utils.sign_pkgs import sign_packages
```